content/docs/integrations/home-page-rss/index.md

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
This whole file talks about the markdown homepage plugin but it is in the home-page-rss folder, is it an accidental copy paste?

Reply:
I didnt mean to add this one. Ive removed it